embed/config.go

```diff
@@ -98,6 +99,10 @@ type Config struct {
 
 	// ForceNewCluster starts a new cluster even if previously started; unsafe.
 	ForceNewCluster bool `json:"force-new-cluster"`
+
+	// ClientHandlers is for registering own handlers.
+	// You must ensure the route paths can't be conflicted with etcd's.
+	ClientHandlers map[string]http.Handler `json:"-"`
```

Comment:
Also mention that this is only designed for embedding etcd into other applications? So we ignore its json representation?